public/wp-content/themes/ColoredCow/src/scss/style.scss

```diff
@@ -34,4 +34,40 @@ Text Domain: coloredcow
 	}
 }
 
+.btn:hover{
+	cursor: pointer;
+}
+
+.images_carousel{
+	position: center;
+	display: block;
+	width:765px;
```

Comment:
does these hard coded height and width also works for mobile? how does the carousel look in mobile? If this is an issue, let's do it mobile first. Read about media queries.